src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -79,8 +79,7 @@ public void createStudentCascadeWithoutDocument(
     
     public void createStudentCascade(StudentAttributes studentData, boolean hasDocument) 
             throws InvalidParametersException, EntityAlreadyExistsException, EntityDoesNotExistException {
-        
-        createStudentCascadeWithSubmissionAdjustmentScheduled(studentData, hasDocument);
+        studentsDb.createStudent(studentData, hasDocument);
```

Comment:
The cascading was done for evaluation submissions, to add a submission to/from the other peers in the same team. For feedback sessions, we currently simply treat this case as a case where there is no response from/to the new student in the team. This does not change the calculations for the values (Claimed contribution and perceived contribution) in contribution questions, as the calculations will ignore the students without a response.